### DISCUSSION TITLE
Data month

### DISCUSSION THREAD

**Discussion by keilebD** [skills/.github#191] [General]
I never know how to use Github but with the step by step of Microsoft i learned more.

**Comment by JuanRoncancioP** [comment]
Incredible course, everything was very well explained for free. Thanks Microsoft

**Comment by deyvi2023** [comment]
el curso esta muy interactivo es bueno que se implemente la practica a todo, gracias

**Comment by AngelgarciaJ** [comment]
Muy bueno, claro y preciso. 👍

**Comment by MahoMarchan** [comment]
Buen curso, claro y practico
